### PR TITLE
Auto close help thread(s) on inactivity

### PR DESCRIPTION
**Pull request by Soheab**
This PR implements a background task that every 5 minutes checks all active threads on activity. Here is how it works:
```py
if not list_of_message_sent_by_the_author_in_15_minutes:
    # await close thread.
elif last_message_is_over_one_day:
    # await send a message in thread mentioning the author.
elif last_message_is_over_two_days
    # await close thread.
```
This has not been tested but it should technically work.

### CODE REVIEW

**Review by rf20008 — APPROVED** [review]
LGTM

**Review by VincentRPS — APPROVED** [review]
nice

**Review by VincentRPS — APPROVED** [review]
nice

**Review by Skelmis — CHANGES_REQUESTED** [review]
Can you run black over these to follow pep8 please.

**Review by Skelmis — APPROVED** [review]
Sorry about that, was tripping on ``=`` gaps on mobile

File: cogs/help.py [hunk 1]
```diff
@@ -200,6 +223,46 @@ async def fake_send(_self, *args, **kwargs):
         FakeContext.send = fake_send
         await self.close(FakeContext(thread, thread_author, thread.guild))
 
+    
+    @tasks.loop(minutes=5.0)
```
Comment (TAG-Epic) [inline cogs/help.py:227]: This should probably be upped quite a bit?

**Review by TAG-Epic — CHANGES_REQUESTED** [review]

File: cogs/help.py (hunk 1 above)
Comment (ooliver1) [reply cogs/help.py:227]: it does that 15m check remember

**Comment by VincentRPS** [comment]
@TAG-Epic

**Pull request closed without merging**